src/RadarScatter.js

```diff
@@ -266,7 +265,6 @@ var RadarScatter = function() {
                 // Customize the chart for being tiny!
                 .lineThickness(0.5)
                 .showVertice(false)
-                .showTooltip(false)
```

Comment:
If there's no `.showTooltip(false)` here, are there tooltips showing up on the tiny versions?